Multisig/UI/Transactions/TransactionDetails/TransactionConfirmationsView/TransactionConfirmationCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  TransactionConfirmationCell.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/10/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionConfirmationCell: View {
+    let address: String
+    let style: TransactionConfiramtionStatusViewStyle
+
+    var body: some View {
+        VStack (alignment: .leading, spacing: 0) {
+            TransactionConfiramtionStatusView(style: style)
+
+            HStack {
+                if [TransactionConfiramtionStatusViewStyle.canceled, TransactionConfiramtionStatusViewStyle.executed].contains(style) {
```

Comment:
please extract this to a named computed variable on the `TransactionConfiramtionStatusViewStyle` itself that would return true/false

Reply:
don't you think this will be specific thing , I mean it used only one time

Reply:
this is for readability, if there's no logical name for it, I'd create then a variable on this view with the name like `hasVerticalBar`